scripts/zones/Arrapago_Remnants/instances/arrapago_remnants.lua

```diff
@@ -2,7 +2,7 @@
 -- Salvage: Arrapago Remnants
 -----------------------------------
 local ID = require("scripts/zones/Arrapago_Remnants/IDs")
-require("scripts/globals/instance")
+local instance_helpers = require ("scripts/globals/instance")
```

Comment:
We had a discussion about this in the devs chat; I think we want to only use the Lua module import pattern for more specific globals, like Zone IDs or Zone-specific helpers. For content coming from real globals (scripts/globals) we want to attach the functionality to the tpz object:

```lua
-- instance.lua

tpz = tpz or {}
tpz.instance = {}

tpz.instance.updateInstanceTime(...)
end

---
-- instances/arrapago_remnants.lua 

instance_object.onInstanceTimeUpdate = function(instance, elapsed)
    updateInstanceTime(instance, elapsed, ID.text)
    tpz.instance.updateInstanceTime(instance, elapsed, ID.text)
end
```